### PR TITLE
refactor: Add check-style rules to avoid reviewers to look for these

### PR DESCRIPTION
**Pull request by sabahirfan**
### JIRA link (if applicable) ###
N/A

### Change description ###
This PR adds check stlye rules to avoid reviewers to look for these and have these caught up early.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by ckalista — APPROVED** [review]

File: config/checkstyle/checkstyle.xml [hunk 1]
```diff
@@ -26,7 +26,12 @@
   <module name="FileTabCharacter">
     <property name="eachLine" value="true"/>
   </module>
-
+  <module name="RegexpMultiline">
+    <property name="format" value="^([^\n ]+ )*(class|interface) [^{]*\{\n[^\n}]"/>
+    <property name="message" value="Leave empty line after class/interface definition!"/>
```
Comment (swalker125) [inline config/checkstyle/checkstyle.xml:31]: hmm I don't personally agree with this. Thoughts?

Comment (sabahirfan) [reply config/checkstyle/checkstyle.xml:31]: I have added this after few review comments to add new line in my recent PR :D

Comment (swalker125) [reply config/checkstyle/checkstyle.xml:31]: I have been saying to remove lines 😂 Sounds like we need to catch up as team

Comment (sabahirfan) [reply config/checkstyle/checkstyle.xml:31]: As discussed offline, let's leave it there to have a consistent code base.

**Review by dharmendrak — APPROVED** [review]

**Pull request merged**